chore: add 9.1.0 alpha release notes (#21576)
OpenTrons/opentrons-api: app-shell/build/release-notes.md

## Opentrons App Changes in 9.1.0

Welcome to the v9.1.0 release of the Opentrons App! This release introduces an Opentrons App version specifically for use with the Flex and includes other new features, improvements, and bug fixes.

> Download the Opentrons OT-2 App [here][https://opentrons.com/app].

Reviewer: I want this to be even stronger.

```suggestion
Welcome to the v9.1.0 release of the Opentrons App! This release is designed specifically for use with Opentrons Flex and includes other new features, improvements, and bug fixes.

**OT-2 robots will no longer appear in the Opentrons App.** v9.1.0 will prompt you to get the separate Opentrons OT-2 App, or you can download it directly from <https://opentrons.com/app>.
```

Author: I like this much better. Thanks for the suggestion!!

### New Features

- Use the updated Opentrons App to control and work with your Flex, including running protocols. Download the OT-2 App to control the OT-2 liquid handling robot.
- View step groups added to your protocols in [visualization][https://docs.opentrons.com/flex/opentrons-app/protocol-viz/].

### Improvements

> - View empty tip racks on the deck map during run setup.

Reviewer: "See whether a tip rack has been marked as empty…" ?

For some reason "View empty tip racks" sounds like you'll see spent tip racks that just happen to be lying around.

Author: Good call. I changed this to `View tip racks marked as empty on the deck map...` so it still implies that when you set the tip rack as empty, you're going to see it in setup

### Bug Fixes

- View individual labware loaded off-deck to check liquid volumes and locations.
- Protocol run setup correctly displays the number of labware loaded in a Flex Stacker.
- Slot spotlights in protocol visualization correctly display the number of tips returned to an empty tip rack.

> - Open slot spotlights for deck slots containing a Flex Stacker without crashes in the app. 

Reviewer: ```suggestion
- Slot spotlights for deck slots containing a Flex Stacker no longer crash. 
```
